### DISCUSSION TITLE
Strange message colors when sensor age exceeds 10 days

### DISCUSSION THREAD

**Discussion by ANDREAPIPPI** [NightscoutFoundation/xDrip#2985] [General]
AAs shownn in the image, the message "Ends in 13.6 days" has white text on a black background, but when the age goes past 10 days, the message changes to "Expires Sun, 3:58pm", in terrible white text on a cream background, which turns out to be almost illegible.
Is this the rule or have I chosen wrong colors in the settings?
I searched but couldn't find my mistake.
Thank you.
Andrew :-)

![strange-colors](https://github.com/NightscoutFoundation/xDrip/assets/32884006/cf6f9da5-8ad8-4a20-8a52-710b3a9539d3)

**Comment by Navid200** [comment]
Thanks for this.
This is actually the second time someone has brought this up.

Let us have a look and see if something can be done.
Would you please remind me if I don't get back to you in two weeks?

**Comment by ANDREAPIPPI** [comment]
Thank you Navid, I'll remind you this, if needed.
Bye

**Comment by ANDREAPIPPI** [comment]
Hello Navid. 
As you requested, I remind you this discussion.

**Reply by Navid200** [reply]
Please click (tap) on create issue from discussion in the right pane.

**Comment by ANDREAPIPPI** [comment]
Thanks, I just opened issue #3052.
Can you please close this discussion?
Thank you very much

**Reply by Navid200** [reply]
There is no need to close a discussion.  It is a discussion, a conversation.  It can be left open.

**Comment by ANDREAPIPPI** [comment]
Thanks, didn't know it.
